frontend/src/app/modules/auth/components/sign-in-form/sign-in-form.component.ts

```diff
@@ -27,7 +26,7 @@ export class SignInFormComponent {
     );
 
     // eslint-disable-next-line no-empty-function
-    constructor(private authService: AuthService, private router: Router) {}
+    constructor(private authService: AuthService, private router: Router, private spinnerService: SpinnerService) {}
```

Comment:
Please delete eslint-disable-next-line

Reply:
But it will cause linter problems with empty constructor. This component is not inherited from base component and the body of constructor is empty

Reply:
So, you need to add BaseComponent

Reply:
Fixed